packages/daimo-common/src/time.ts

```diff
@@ -41,6 +59,10 @@ export function timeString(s: number) {
   });
 }
 
+export function dateStringToUnixSeconds(date: string): number {
+  return Math.floor(new Date(date).getTime() / 1000);
```

Comment:
what kinds of string does this expect? the `Date` constructor is unfortunately locale dependent

![Uploading image.png…]()

Reply:
i think i'll just change everything to a unix timestamp on the landline side. i used `Date` objects and it's been annoying